app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_name/AddBusStopNameForm.kt

```diff
@@ -15,11 +19,25 @@ class AddBusStopNameForm : AAddLocalizedNameForm<BusStopNameAnswer>() {
     override val addLanguageButton get() = binding.addLanguageButton
     override val namesList get() = binding.namesList
 
+    override val adapterRowLayoutResId = R.layout.quest_localizedname_row
+
     override val otherAnswers = listOf(
         AnswerItem(R.string.quest_placeName_no_name_answer) { confirmNoName() },
         AnswerItem(R.string.quest_streetName_answer_cantType) { showKeyboardInfo() }
     )
+    private val busStopNameSuggestionsSource: BusStopNameSuggestionsSource by inject()
 
+    override fun getLocalizedNameSuggestions(): List<List<LocalizedName>> {
+        val polyline = when (val geom = geometry) {
+            is ElementPolylinesGeometry -> geom.polylines.first()
+            is ElementPolygonsGeometry -> geom.polygons.first()
+            is ElementPointGeometry -> listOf(geom.center)
+        }
+        return busStopNameSuggestionsSource.getNames(
+            listOf(polyline.first(), polyline.last()),
+            MAX_DIST_FOR_BUS_STOP_NAME_SUGGESTION
+        )
```

Comment:
why?

Reply:
removed in [86bd839](https://github.com/streetcomplete/StreetComplete/pull/6097/commits/86bd839b8d5aca7e0909fabdd73b9f8b34d9c6a6)